test/e2e/llmisvc/test_llm_inference_service.py

```diff
@@ -12,23 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
 import time
-from dataclasses import dataclass
-from typing import Any, Callable, List
+
+import os
 import pytest
 import requests
-from kubernetes import client
 import yaml
+from dataclasses import dataclass
 from kserve import KServeClient, V1alpha1LLMInferenceService, constants
+from kubernetes import client
+from typing import Any, Callable, List
+
 from .diagnostic import (
     print_all_events_table,
     kinds_matching_by_labels,
 )
 from .fixtures import (
     generate_test_id,
     # Factory functions are not called explicitly, but they need to be imported to work
-    test_case,  # noqa: F401,F811
+    test_case,
+    inject_k8s_proxy,  # noqa: F401,F811
```

Comment:
nit: shouldn't we do it like this?
```suggestion
    test_case, # noqa: F401,F811
    inject_k8s_proxy,  
```

AFAICT you call `inject_k8s_proxy()` explicitly.